src/Command/CommandBase.php

```diff
@@ -241,6 +241,11 @@ private function setLocalDbHost(): void {
     if ($lando_info = self::getLandoInfo()) {
       $this->localDbHost = $lando_info->database->hostnames[0];
     }
+    // If code studio pipeline.
+    else if (!empty(getenv('CI_JOB_ID')) && !empty(getenv('ACQUIA_GLAB_TOKEN_NAME'))) {
+      $this->localDbHost = 'mysql';
+    }
```

Comment:
I tried to add test for this and seems it will be much like `PullDatabaseCommandTest::testPullDatabasesIntoLando()` with only change in mysql host (and env variable set). But then I tried to change some code for testPullDatabasesIntoLando() like changed the db host to `localhost`, also in LandoInfoHelper but looks like tests are always passing. So not sure if relevant to add a test for ths as that will be just a copy paste